tests/postman/ElectionGuard API Tests.postman_collection.json

```diff
@@ -1052,15 +1103,280 @@
+											"pm.test(\"Ballots are returned in the same order as the request\", function () {",
+											"    const requestJson = JSON.parse(pm.request.body.raw)",
+											"    const responseJson = pm.response.json();",
+											"    pm.expect(responseJson.encrypted_ballots.length).to.eql(requestJson.ballots.length)",
+											"    for (let i = 0; i < requestJson.ballots.length; i++) {",
+											"        requestBallot = requestJson.ballots[i]",
+											"        responseBallot = responseJson.encrypted_ballots[i]",
+											"        pm.expect(responseBallot.object_id).to.eql(requestBallot.object_id)",
+											"    }",
+											"});",
+											"",
+											"pm.test(\"Ballots use proper code seed hashes\", function () {",
+											"    const requestJson = JSON.parse(pm.request.body.raw);",
+											"    const responseJson = pm.response.json();",
+											"    latest_hash = requestJson.seed_hash;",
+											"",
+											"    for (const ballot of responseJson.encrypted_ballots) {",
+											"        pm.expect(ballot.code_seed).to.eql(latest_hash);",
+											"        latest_hash = ballot.crypto_hash;",
+											"    }",
+											"    pm.expect(responseJson.next_seed_hash).to.eql(latest_hash);",
+											"});",
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n  \"election_id\": \"{{e2e-election-id}}\",\n  \"seed_hash\": \"B52C5D5946B74F45001E90F5D9BED765C52921F0ED11A5F430F50CDF82EE4756\",\n  \"ballots\": [\n    {\n      \"object_id\": \"ballot-1663ab54-e95f-11eb-bd0c-acde48001123\",\n      \"style_id\": \"congress-district-7-lacroix\",\n      \"contests\": [\n        {\n          \"object_id\": \"president-vice-president-contest\",\n          \"ballot_selections\": [\n            {\n              \"is_placeholder_selection\": false,\n              \"object_id\": \"barchi-hallaren-selection\",\n              \"vote\": 1\n            },\n            {\n              \"is_placeholder_selection\": false,\n              \"object_id\": \"cramer-vuocolo-selection\",\n              \"vote\": 0\n            }\n          ]\n        }\n      ]\n    },\n    {\n      \"object_id\": \"ballot-1663ab54-e95f-11eb-bd0c-acde48001123\",\n      \"style_id\": \"congress-district-7-lacroix\",\n      \"contests\": [\n        {\n          \"object_id\": \"president-vice-president-contest\",\n          \"ballot_selections\": [\n            {\n              \"is_placeholder_selection\": false,\n              \"object_id\": \"barchi-hallaren-selection\",\n              \"vote\": 0\n            },\n            {\n              \"is_placeholder_selection\": false,\n              \"object_id\": \"cramer-vuocolo-selection\",\n              \"vote\": 1\n            }\n          ]\n        }\n      ]\n    }    \n  ]\n}\n",
```

Comment:
Two cast ballots, one for each candidate.  No spoiled ones yet, I'll worry about that later.